src/material-experimental/mdc-progress-spinner/progress-spinner.ts

```diff
@@ -95,8 +95,11 @@ export class MatProgressSpinner extends _MatProgressSpinnerMixinBase implements
     hasClass: (className: string) => this._elementRef.nativeElement.classList.contains(className),
     removeClass: (className: string) => this._elementRef.nativeElement.classList.remove(className),
     removeAttribute: (name: string) => this._elementRef.nativeElement.removeAttribute(name),
-    setAttribute: (name: string, value: string) =>
-      this._elementRef.nativeElement.setAttribute(name, value),
+    setAttribute: (name, value) => {
+      if (name !== 'aria-valuenow') {
```

Comment:
nit: maybe we should have a comment here explaining why we're special-casing it? Also do we have to account for `aria-valuemin` and `aria-valuemax`?

Reply:
aria-valuenmin and aria-valuemax are static and set in the host